### PR TITLE
Customization for AvatarView

### PR DESCRIPTION
**Pull request by buh**
- added open methods for `AvatarView` customization:
  - in `ChannelsViewController`:
```swift
open func updateChannelCellAvatarView(in cell: ChannelTableViewCell, channel: Channel)
```
  - in `ChatViewController`:
```swift
open func updateMessageCellAvatarView(in cell: MessageTableViewCell, message: Message, messageStyle: MessageViewStyle)

open func updateFooterTypingUserAvatarView(footerView: ChatFooterView, user: User)
```
- updated `AvatarViewStyle`: addad new properties:
  - `placeholderTextColor: UIColor?`
  - `placeholderBackgroundColor: UIColor?`

### CODE REVIEW

File: Sources/UI/Views/AvatarView.swift [hunk 1]
```diff
@@ -14,87 +14,130 @@ import Nuke
...
+    public private(set) lazy var avatarLabel: UILabel = {
         let label = UILabel(frame: .zero)
         label.textAlignment = .center
         label.isHidden = true
-        label.preferredMaxLayoutWidth = 2 * layer.cornerRadius
+        label.preferredMaxLayoutWidth = 2 * style.radius
         return label
     }()
     
     /// Create a AvatarView with a given corner radius.
```
Comment (b-onc) [inline Sources/UI/Views/AvatarView.swift:27]: Needs updating

File: CHANGELOG.md [hunk 2]
```diff
@@ -23,6 +23,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added `ChatViewStyle.default`. 
 - Added `ChatViewStyle.dynamic` for iOS 13 to support dynamic colors for dark mode.
 - Added `MessageViewStyle.pointedCornerRadius` to make pointed corner rounded.
+- Added methods for `AvatarView` customization:
```
Comment (b-onc) [inline CHANGELOG.md:26]: Needs PR link (203)

File: Sources/UI/Views/AvatarView.swift (hunk 1 above)
Comment (buh) [reply Sources/UI/Views/AvatarView.swift:27]: What do you mean?

Comment (b-onc) [reply Sources/UI/Views/AvatarView.swift:27]: The initializer changed, no longer accepts a corner radius

**Comment by buh** [comment]
CHANGELOG updated.

File: Sources/UI/Views/AvatarView.swift (hunk 1 above)
Comment (b-onc) [reply Sources/UI/Views/AvatarView.swift:27]: I mean the documentation
> /// Create a AvatarView with a given corner radius.

needs updating to
> /// Create an AvatarView with a given AvatarViewStyle

Comment (buh) [reply Sources/UI/Views/AvatarView.swift]: Fixed.

**Comment by buh** [comment]
Anything else for this PR?

**Pull request merged**